securedrop/source_app/utils.py

```diff
@@ -61,11 +63,10 @@ def generate_unique_codename(config):
             return codename
 
 
-def get_entropy_estimate():
+def get_entropy_estimate() -> int:
     with io.open('/proc/sys/kernel/random/entropy_avail') as f:
         return int(f.read())
 
-
 def asynchronous(f):
     def wrapper(*args, **kwargs):
         thread = Thread(target=f, args=args, kwargs=kwargs)
```

Comment:
Not sure how to annotate these functions...

Reply:
there seems less value to adding static type annotations to this specific function so feel free to leave out, but if you want to go ahead:  `typing.Any` for args and kwargs seems reasonable here (since it's _such_ a generic function, if anyone disagrees here feel free to), and since this is a decorator (i.e. a function that wraps a function), type of `f` is `typing.Callable`